src/api.ts

```diff
@@ -954,14 +954,53 @@ function createApi({
         throw new NotFoundError(id)
       }
 
-      // TODO
-      // - Delete any orphaned tilesets and sprites
-      // - How to handle glpyhs here?
+      // TODO: How to handle glpyhs here?
       const deleteStyleTransaction = db.transaction(() => {
         db.prepare(
           'DELETE FROM Import WHERE areaId IN (SELECT id FROM OfflineArea WHERE styleId = ?)'
         ).run(id)
```

Comment:
I don't think we want to delete anything from the import table? I guess we could, but it's fine I think to keep that record in there.

Reply:
what about the offline areas table? thought it would make sense to delete there, but in order to do that, we'd need to either delete the associated records in the import table, or update the import table to dereference the offline areas

Reply:
Yes, good point, let's delete the imports too.